src/EventStore.Client.Streams/StreamMetadataResult.cs

```diff
@@ -2,24 +2,74 @@
 
 #nullable enable
 namespace EventStore.Client {
+	/// <summary>
+	/// Represents stream metadata as a series of properties for system
```

Comment:
@thefringeninja  I didn't understand this sentence (the system data part) :( Could you please clarify the meaning?

Reply:
Same

Reply:
I just copied this from the TCP client. I'll add examples